src/Razor/test/Microsoft.CodeAnalysis.Razor.Test/ComponentTagHelperDescriptorProviderTest.cs

```diff
@@ -1436,5 +1436,72 @@ public class MyComponent : MyBase
 
             Assert.Empty(component.BoundAttributes);
         }
+
+        [Fact] // Testing multilevel overrides with the [Parameter] attribute on different levels.
+        public void Execute_MultiLevelOverriddenProperties_CreatesDescriptorCorrectly()
```

Comment:
Added just one test but it covers a bunch of scenarios. We already have a test for properties with `new` hiding.